aea/helpers/dialogue/base.py

```diff
@@ -829,24 +879,16 @@ def _create_opponent_initiated(
         )
 
         assert dialogue_label not in self.dialogues
-        dialogue = self.create_dialogue(dialogue_label=dialogue_label, role=role)
+        dialogue = self._dialogue_class(
+            dialogue_label=dialogue_label,
+            message_class=self._message_class,
+            agent_address=self.agent_address,
+            role=role,
+        )
         self.dialogues.update({dialogue_label: dialogue})
 
         return dialogue
 
-    @abstractmethod
-    def create_dialogue(
```

Comment:
Let's keep the method (so we keep the interface definition) until 0.6 but remove the decorator. You can also add a note that it is deprecated...

Reply:
Done